ReleaseNotes.md

```diff
@@ -6,35 +6,73 @@
 
 ## What's New in 3.0.0?
 
-Currently the contents of this file represent a placeholder for future Infusion 3.0.0 release notes.
-
 ### New Features
 
+* Build
+  * Minified distributions:
+    * infusion-all.js
+    * infusion-all-no-jquery.js
+    * infusion-framework.js
+    * infusion-framework-no-jquery.js
+    * infusion-uio.js
+    * infusion-uio-no-jquery.js
+* Framework
+  * Added model transformations for converting between:
+    * Boolean values and Strings
+    * Date/Time and Strings
+    * JSON Objects and Strings
+  * Updated model transformations:
+    * Number to String transformation supports specifyhing decimal precission.
+    * Round tranformation can round to an integer or decimal value
+  * `fluid.stringTemplate` updated to support nested objects for interpolate values
+  * Added `fluid.dataSource` grade
+  * Added `fluid.remoteModelComponent` for keeping remote and local models in sync.
 * Preference framework
-  * Updated look of on/off toggles
+  * Switched from Stylus to SASS for CSS pre-processing
+  * Responsive design for small screens/mobile devices.
+  * Updated look of on/off toggles and checkboxes
+  * Added additional contrast themes based on Windows contrast themes.
   * Added the [OpenDyslexic 3](https://opendyslexic.org/) font as an option to the Text Style panel
-
-### Removal of Deprecated Features
-
-* Preference framework
-  * Collapsed inputsLarger and emphasizeLinks preferences into enhanceInputs
+  * Added localized message bundles for Farsi, French, Portugese, and Spanish.
+  * New preferences:
+    * Letter spacing
+    * Syllabification preference
+    * Text-to-speech preference using the Orator component
+    * Word spacing preference
+* Orator
+  * A self voicing widget with play/pause, text highlighting, selection reading.
```

Comment:
The problems with the Chrome TTS implementation that prevent this from currently being usable are probably serious enough to warrant mentioning here